update.py

```diff
@@ -686,7 +692,7 @@ def cache_parameters_files(site=args.site):
     old_params_mversion/ folders and .html built files as well.
     """
     for key, value in PARAMETER_SITE.items():
-        if (site == key or site is None) and (key != 'AP_Periph'):  # and (key != 'AP_Periph') workaround until create a versioning for AP_Periph in firmware server
+        if (site == key or site is None) and (key not in ['AP_Periph', 'heli']):  # and (key != 'AP_Periph') workaround until create a versioning for AP_Periph in firmware server
```

Comment:
Is it still the versioning system on the Wiki server that I did in 2019? If so, I'd be happy to add these two versionings when time permits. let me know. ping @Hwurzburg and ping @peterbarker

Reply:
@brunoolivieri Bruno, AFAIK its the same system....any help would be appreciated...